src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/legaladvisor/OrderPostProcessor.java

```diff
@@ -65,6 +65,10 @@ public CallbackResponse copyDraftToCaseDocument(CallbackParams callbackParams) {
         HearingCourt hearingCourt = directionOrderService.getHearingCourt(ccdCase);
 
         CCDCase updatedCase = ccdCase.toBuilder()
+            .expertReportPermissionPartyGivenToClaimant(null)
```

Comment:
Do we necessarily need to remove these field? looks like we are clearing the data provided.

Reply:
We do need to clear these down as the CCD definition has changed. However we are translating and capturing the information in OrderCreator

Reply:
We do need to clear these down as the CCD definition has changed. However we are translating and capturing the information in OrderCreator

Reply:
thanks @daveoh